### PR TITLE
Update app-templates.md

### PR DESCRIPTION
**Pull request by YogeshIITK**
Adding following app template descriptions :
1- Training App Template
2- Quiz App Template

### CODE REVIEW

File: msteams-platform/samples/app-templates.md [hunk 1]
```diff
@@ -608,6 +621,19 @@ A project can include multiple tasks, and various projects can be assigned to em
 ![Time Tally](../assets/images/11zon_gif.gif)
 
 
+## Training  &#9734;
+
+Training is a custom [Teams messaging
+extension ](../messaging-extensions/what-are-messaging-extensions.md) app that enables customers to publish a training within a chat or a channel for offline knowledge sharing and upskilling. The app is supported across all Teams platform clients - desktop, browser, iOS, and Android and is ready for deployment as part of your Microsoft 365 subscription.
```
Comment (v-pmuthyala) [inline msteams-platform/samples/app-templates.md:627]: customers to publish training

File: msteams-platform/samples/app-templates.md [hunk 2]
```diff
@@ -467,6 +467,19 @@ Quick Responses is a Microsoft Teams app that delivers a robust solution for eff
 
 ![Sample view of responses](../assets/images/quick-responses.png)
 
+
+## Quiz  &#9734;
+
+Quiz is a custom [Teams messaging extension ](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to create a quiz within a chat or a channel for knowledge check and instantaneous results. Use Quiz for – In-class and offline exams, Knowledge check within team and for fun quizzes within team. Quiz app is supported across all platforms – Teams desktop, browser, iOS, and Android clients. It is ready for deployment as part of your existing Microsoft 365 subscription.
```
Comment (v-pmuthyala) [inline msteams-platform/samples/app-templates.md:473]: knowledge check within the team, and for fun quizzes within the team.

**Review by v-pmuthyala — CHANGES_REQUESTED** [review]
minor changes

**Review by surbhigupta12 — APPROVED** [review]
Approve the PR.

**Review by KirtiPereira — APPROVED** [review]
Approved.

**Review by v-pmuthyala — APPROVED** [review]
changes reviewed.

**Pull request merged**